test/regressions/index.js

```diff
@@ -1,7 +1,6 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import ReactDOM from 'react-dom';
-import { BrowserRouter as Router, Switch, Route, Link } from 'react-router-dom';
```

Comment:
Why would you remove it here? The e2e suite already has a proper migration

Reply:
https://github.com/mui-org/material-ui/pull/29553#issuecomment-974031464